components/02-molecules/video/video.yml

```diff
@@ -0,0 +1,3 @@
+text_with_video__heading: 'This Is Where A Video Title Will Go'
+text_with_video__text: '<p>This is where captions for videos will go.</p>'
```

Comment:
Make sure you update these two variables to match the changes in the twig.

As a matter of fact, you can probably do a find and replace over your codebase for `text_with_video__` and replace it with just `video__`

Make sure to look through the files it's going to change, but I think it should work.